package.json

```diff
@@ -24,7 +24,7 @@
     "start": "electron app --disable-http-cache --no-electron-connect",
     "reinstall": "node ./tools/reinstall-node-modules.js",
     "postinstall": "electron-builder install-app-deps",
-    "test": "xo",
+    "test": "xo && stylelint app/renderer/css/*.css",
```

Comment:
I think it's important to Windows users that `xo` is run at the end. That test takes care of line endings, which can mess up quite often and I think they're taken care of by post-commit git hooks anyway. So if `xo` fails because of line endings, `stylelint` won't run and we don't want that.

Reply:
hmm... If stylelint fails, would xo run ?

Reply:
No, but then the code has actually failed the test. In the present case, even correctly styled code (which xo says is wrong but passes Travis tests) will not go through stylelint. So I think we should change the order.

Reply:
ok got it.. will do.. thanks :)